dev-libs/libjwt/libjwt-1.15.3.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="JWT C Library"
+HOMEPAGE="https://github.com/benmcollins/libjwt"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/benmcollins/libjwt"
+else
+	SRC_URI="https://github.com/benmcollins/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="MPL-2.0"
+SLOT="0"
+# openssl / gnutls set which ssl implementations to use (build libjwt-ossl.so / libjwt-gnutls.so)
+# IF openssl is enabled it will be the implementation used for libjwt.so
+# gnutls will only be used for libjwt.so if openssl is disabled
+IUSE="gnutls +openssl test"
+
+REQUIRED_USE="
+	|| ( openssl gnutls )
+"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-libs/jansson
+	openssl? (
+		>=dev-libs/openssl-0.9.8:=
+	)
+	gnutls? (
+		>=net-libs/gnutls-3.5.8:=
+	)
+"
+
+DEPEND="
+	${RDEPEND}
+	test? ( dev-libs/check )
+"
+
+PATCHES=(
+	"${FILESDIR}/libjwt-1.15.3_multi_ssl_atools.patch"
+)
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	local myeconfargs=(
+		--enable-multi-ssl
```

Comment:
Should we only pass this if BOTH gnutls and openssl?  It *probably* doesn't matter but could trigger some load-time inefficiency or load-time issue perhaps?

Reply:
The --enable-multi-ssl is always enabled specifically to always create libjwt-ossl.so and/or libjwt-gnutls.so

Background:
This was originally added to the oubliette overlay to support Zoneminder. The Zoneminder developers build on Debian.
Debian has two different packages libjwt and libjwt-gnutls. The stable Zoneminder builds specifically search for libjwt-gnutls.so (well via pkgconfig) when enabling a gnutls build (stable does not use libjwt for openssl). Current development versions compound this by specifically searching for libjwt.so for an openssl build.

I have no idea how many other packages are lurking out in the world depending on Debian's .so naming.

Redhat, Fedora and Arch don't provide a gnutls linked libjwt at all, (so I assume they only support Zoneminder built with openssl).